composer.json

```
@@ -114,7 +114,8 @@
     "config": {
         "process-timeout": 3000,
         "allow-plugins": {
-            "composer/package-versions-deprecated": true
+            "composer/package-versions-deprecated": true,
+            "php-http/discovery": true
```

```suggestion
            "*/*": false
```

Disclaimer: I haven't tested if it works, but I like this direction more.

Hopefully it would both enable installation and save us from other "helpful" 3rd party plugins in the future

(I don't feel like we actually need this new plugin to be enabled)

---

Looks like it works, thanks!